numpy/lib/type_check.py

```diff
@@ -183,6 +194,11 @@ def imag(val):
         return asanyarray(val).imag
 
 
+def _iscomplex_dispatcher(x):
```

Comment:
Next few could be common `_is_type_dispatcher`?

Reply:
done!